h2o-algos/src/main/java/hex/klime/KLimeModel.java

```diff
@@ -27,6 +27,11 @@ public KLimeModel(Key<KLimeModel> selfKey, KLimeParameters params, KLimeOutput o
   }
 
   @Override
+  public GridSortBy getDefaultGridSortBy() {
```

Comment:
Note: This slightly overlaps with AutoML leaderboard sort criterion.
I still do not understand why leaderboard does not extend Grid, but that's discussion for different PR.